controllers/api-controller.js

```diff
@@ -0,0 +1,5 @@
+const endpoints = require("../endpoints.json");
+
+exports.getAllApis = (req, res, next) => {
+  res.status(200).send(endpoints);
```

Comment:
This one is up to you, but I would personally send back the endpoints object as a value on an object with a key like 'endpoints' - it just helps with readability of what we're working with. But up to you 👍